solr_conf/conf/solrconfig.xml

```diff
@@ -76,26 +146,27 @@
 
       <str name="qf">
         sw_display_title_tesim^5
-        sw_author_tesim^3
+        contributor_text_nostem_im^3
         topic_tesim^2
 
-        exploded_project_tag_ssim^2
-        exploded_nonproject_tag_ssim
-        exploded_tag_ssim
         tag_ssim
```

Comment:
tag_ssim is sufficient for searching. "exploded" tag fields are for hierarchical facets